java/ql/src/codeql-suites/java-code-quality.qls

```diff
@@ -11,4 +11,5 @@
       - java/unused-container
       - java/input-resource-leak
       - java/output-resource-leak
-      - java/type-variable-hides-type
\ No newline at end of file
+      - java/type-variable-hides-type
+      - java/do-not-call-finalize
```

Comment:
Ugh, all the PRs are going to have merge conflicts here 😢 . Not sure what we can do about it though. I suppose we could update this file once at the end of porting all the queries. But it's a bit error-prone. If we made the file alphabetically sorted then we'd get fewer conflicts.

Reply:
Yeah, the frequent merge conflicts will be annoying 😭 

> I suppose we could update this file once at the end of porting all the queries. But it's a bit error-prone.

This might be a good idea as long as we do a good job of keeping track of which queries to add, but you're right that it's a bit error-prone.

> If we made the file alphabetically sorted then we'd get fewer conflicts.

This sounds like a good idea for human readability even if we update the file once at the end.